### ISSUE TITLE
"T/F" of PBC when reading ".xyz" datasets

### ISSUE THREAD

**Issue by hghermes** [openkim/kliff#169]
The official manual for Kliff states that the pbc of the dataset can be read as a boolean (True/False) type, but in reality, errors occur when attempting to do so.

**Additional context**
This can be solved by changing the 239 line of "kliff/dataset/extxyz.py 
```
            value = [int(i) for i in value]
```
to 
```
            if(all(i in ['T','F'] for i in value)):
                value = [1 if i =='T' else 0 for i in value]
            else:
                value = [int(i) for i in value]
```

**Comment by mjwen** [comment]
Thanks @hghermes ! 

Do you want to make a PR to fix it?

**Comment by hghermes** [comment]
@mjwen 
No. It's okay, thanks.
I just wanted to share a simple error and fix. Always grateful for you and KLIFF.

**Comment by mjwen** [comment]
Thanks for reporting! Got it fixed in #170
